ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/inventory/host_vars/host2/network_services_keys_name.yml

```diff
@@ -0,0 +1,67 @@
+---
+type: pe
+design:
+  type: mpls
+
+pe:
+  defaults:
+    loopback_ipv4_pool: 10.42.0.0/24
+    bgp_as: 42
+    id: 42
```

Comment:
Please move this to `node_type.yml`

Reply:
we have type `l3leaf` in node_type.yml, did you mean `node_type_keys.yml`

Reply:
This is for host2.

Reply:
ohh, so we should keep a file node_type.yml in host2 with this config?

Reply:
One node_type.yml for host1 and another for host2. This comment is only about the file for host2.

Reply:
Got it!